Gemfile.lock

```diff
@@ -142,7 +144,7 @@ GEM
       rspec-mocks (~> 3.8.0)
       rspec-support (~> 3.8.0)
     rspec-support (3.8.2)
-    sprockets (3.7.2)
+    sprockets (4.0.0)
```

Comment:
Sprockets 4.0 makes some significant changes. I don't know if we are comfortable with making the jump to the next major version yet.